test/tools/scripts/remove-testcases.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+
+import re
+import os
+import sys
+
+
+def main():
+    e2e_path = os.path.dirname(__file__) + "/extracted/libsolidity/semanticTests/end-to-end/"
+    tests = []
+    for f in os.listdir(e2e_path):
+        if f.endswith(".sol"):
+            tests.append(f.replace(".sol", ""))
+
+    cpp_file = open(os.path.dirname(__file__) + "/../../libsolidity/SolidityEndToEndTest.cpp", "r")
+    inside_test = False
+    test_name = "'"
+    ignore = False
+    new_lines = 0
+    for line in cpp_file.readlines():
```

Comment:
Why not do that at the same time as the extraction happens?

Reply:
yes, that would be much nicer